linux/installation/mde_installer.sh

```diff
@@ -790,7 +790,7 @@ onboard_device()
 
     if [[ $ONBOARDING_SCRIPT == *.py ]]; then
         # Make sure python is installed
-        PYTHON=$(which python || which python3)
+        PYTHON=$(which python  2>&- || which python3  2>&-)
```

Comment:
Curious, why did we use 2>&- instead of 2 > /dev/null ?

Reply:
I have actually changed this to 2>/dev/null accommodate shell compatibility concerns. The original use of 2>&- was intended to just dump to a closed file descriptor instead of a defined destination (subtle differences really).